rfcs/20210510-sum-with-zero-point/README.md

```diff
@@ -0,0 +1,60 @@
+# Proposal for support sum with zero-point
+
+## Motivation
+The motivation to provide such support is to enable dequantize the 
+asymmetrically quantized sum's src1 tensor to f32 domain before performing the 
+sum operation. In general we want to support following operation:
+`dst[:] := scale * (dst[:] - zero_point) + op(...)`
```

Comment:
We should consider multiple zero points because lack of that support will introduce inconsistency in zero points support and will complicate life of users. It is easier/more scalable to support the same quantization across all functionality. Also it will be hard to extend the API later ones users request multiple zero points support.

Reply:
I think this RFC is mostly about the sum postop. As for now, we support only scalar scale in postops.
Supporting multiple multiple scale/zero-points for postops might be useful (did we have any request for that? ).
However I would suggest to keep that for later as we would need to use the same mechanism for all postops (e.g. using execution arguments (`DNNL_ARG_ATTR_MULTIPLE_POST_OP(idx) | DNNL_ARG_ZERO_POINT`).

Reply:
Generally, if we support here multiple zero points it will be inconsistent with scale in `append_sum`. So maybe until we have also single scale value and no request for user, we can stay with this single zero-point value too?

Reply:
thanks @mgouicem and @bartekxk for your answers,
my initial thought was we support multiple zero points in convolution src/dst, so sum operation between such convolutions has to support multiple zero points as well. Agree, that in terms of post operations everything supports only single scale and it makes sense to keep sum consistent with that part...
@bartekxk , Could you please clarify what is the initial issue you are trying to address? If there is no need in multiple zero points right now I am fine to go with this proposal as is.

Reply:
@igorsafo Initial issue is "[Pytorch] Request conv sum post-op to support asymmetric quantization".